tensorflow/lite/python/optimize/calibrator_test.py

```diff
@@ -80,7 +80,7 @@ def input_gen():
 
   def test_invalid_model_buffer(self):
     float_model = b'\0' * 100
-    with self.assertRaisesWithRegexpMatch(ValueError,
+    with self.assertRaisesRegex(ValueError,
                                           'Failed to parse the model'):
```

Comment:
thanks could you also fix the formatting here, now that the indent has changed.

Reply:
Thanks for the feedback. Done.